float64 camino_x
float64 camino_y